chat-v2-sample/CocoaPods/Podfile

```diff
@@ -6,8 +6,8 @@ target 'swift-chat-v2-sample' do
   use_frameworks!
 
   # Pods for swift-chat-v2-sample
-  pod 'KustomerChat', '~> 5.0.2'
-  # UNCOMMENT THIS IF TESTING FROM BRANCH: pod 'KustomerChat', :git => 'https://github.com/kustomer/kustomer-ios.git', :branch => 'releases/5.0.2'
+  pod 'KustomerChat', '~> 5.0.3'
+  # UNCOMMENT THIS IF TESTING FROM BRANCH: pod 'KustomerChat', :git => 'https://github.com/kustomer/kustomer-ios.git', :branch => 'releases/5.0.3'
```

Comment:
Should this be committed to the repo?

Reply:
@emre-ozsahin it has been there historically - I can remove it.